PendingReleaseNotes.md

```diff
@@ -37,6 +37,7 @@
   Ceph version which Rook has detected in the pod(s) being run by the resource.
 - OSDs provisioned by `ceph-volume` now supports `metadataDevice` and `databaseSizeMB` options.
 - The flex driver can be configured to properly disable SELinux relabeling and FSGroup with the settings in operator.yaml.
+- Rook can now be configured to read "region" and "zone" labels on Kubernetes nodes and use that information as part of the CRUSH location for the OSDs.
```

Comment:
You'll need to rebase to get the new (mostly empty) pending release notes since 1.0 was released.